src/atoms/modal/modal.tsx

```diff
@@ -28,16 +28,24 @@ export const ModalBody = Styled(Card)<ModalCardProps>`
   gap: 4px; 
   max-width: ${({ maxWidth }) => maxWidth ?? '600px'};
   height: ${({ height }) => height ?? '62vh'};
-  max-height: ${({ maxHeight }) => maxHeight ?? '62vh'};
+  max-height: ${({ maxHeight }) => maxHeight ?? '600px'};
   box-shadow: ${({ theme }) => theme.dropShadow.REGULAR};
-  max-height: 600px;
+
+  ${tablet(css`
+    max-width: calc(100vw - 208px);
```

Comment:
based on design this should happen on mobile as well but i tested it on my phone and it was way too thin and previously it looked perfect imo.